controllers/controllers/clusters/vsphere.go

```diff
@@ -110,8 +112,19 @@ func (v *VSphereClusterReconciler) bundles(ctx context.Context, name, namespace
 	return clusterBundle, nil
 }
 
+func (v *VSphereClusterReconciler) eksdRelease(ctx context.Context, name, namespace string) (*eksdv1alpha1.Release, error) {
```

Comment:
Sure. This change will affect the `BuildSpecForCluster` method. Since refactoring that is a follow-up task here, I would prefer to make that change in that separate PR